src/sentry/services/hybrid_cloud/integration/service.py

```diff
@@ -147,19 +149,32 @@ def get_organization_context(
 
     @rpc_method
     @abstractmethod
-    def get_organization_contexts(
```

Comment:
I might be reading this wrong, is this safe to delete? I'm still seeing usages of this service method

Reply:
No, this is wrong. I meant to add the new signatures only.